EasyPost/Parameters/CarrierAccount/Create.cs

```diff
@@ -8,7 +10,7 @@ namespace EasyPost.Parameters.CarrierAccount
     ///     <a href="https://www.easypost.com/docs/api#create-a-carrier-account">Parameters</a> for <see cref="EasyPost.Services.CarrierAccountService.Create(Create, System.Threading.CancellationToken)"/> API calls.
     /// </summary>
     [ExcludeFromCodeCoverage]
-    public class Create : BaseParameters<Models.API.CarrierAccount>, ICarrierAccountParameter
+    public class Create : ACreate
```

Comment:
question: Why `ACreate`? Seems like a typo? If not can we instead define what this is better as `A` doesn't make sense alone?

Reply:
`A` as in abstract, the same way `I` as in interface.

Reply:
Ah, ok - thanks!